### ISSUE TITLE
optional column "datalink_url" cannot be found

### ISSUE THREAD

**Issue by Laubeee** [epn-vespa/DaCHS-for-VESPA#9]
Hi, it seems the optional column `datalink_url` as [documented here](https://voparis-wiki.obspm.fr/display/VES/EPN-TAP+v2+parameter+description) does not work. When I try to use it:

```xml
<table id="epn_core" onDisk="True" adql="True">
  <mixin spatial_frame_type="none" optional_columns="datalink_url time_scale thumbnail_url access_estsize" >//epntap2#table-2_0</mixin>
</table>
```

it errors with `'datalink_url' could not be located in table optional_columns`

As a workaround I defined the column myself with:
```xml
<column name="datalink_url" type="text" ucd="meta.ref.url;meta.datalink" description="Link to folder with observation fits files" />
```

solutions are to either update the documentation or the optional columns

Cheers

**Comment by Erard** [comment]
That's right. You'll find  (old) examples of datalink in EPN-TAP service here
https://github.com/epn-vespa/DaCHS-for-VESPA/blob/master/q.rd_examples/spicam/q.rd_spicam
https://github.com/epn-vespa/DaCHS-for-VESPA/blob/master/q.rd_examples/hst/HST_q.rd

**Comment by BaptisteCecconi** [comment]
Hello @Laubeee, indeed the `datalink_url` is not yet implemented in the `mixin` The column must be implemented manually, at this point. 

However, the use of this optional column should be described in more details, since it is not in the _Datalink_ standard.

**Comment by Laubeee** [comment]
Small detail: is there anything that makes the ADQL query form result display the column as an actual URL, like it does with access_url?

**Comment by Erard** [comment]
You mean in DaCHS? I think not. It actually links to a page built on the fly. You can experiment with TOPCAT, which has internal support for datalink, or with the VESPA portal.
